examples/with-tailwind/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to Remix!
```

Comment:
Did you create this copying the template folder or running the Remix CLI?

Reply:
@sergiodxa I ran Remix CLI for this.

Reply:
@sergiodxa Is there any issue you want me to resolve?